example/sofast_fringe/example_calibration_screen_shape.py

```diff
@@ -13,30 +13,41 @@
 import opencsp.common.lib.tool.log_tools as lt
 
 
-def run_screen_shape_calibration(save_dir):
+def example_screen_shape_calibration():
     """Runs screen shape calibration. Saves a DisplayShape HDF5 file
     to ./data/output/screen_shape/display_shape.h5
     """
-    # Load output data from Scene Reconstruction (Aruco marker xyz points)
-    file_pts_data = join(
-        opencsp_code_dir(), 'common/lib/deflectometry/test/data/data_measurement', 'point_locations.csv'
+    # Define save directory
+    dir_save = join(dirname(__file__), 'data/output/screen_shape')
+    ft.create_directories_if_necessary(dir_save)
+
+    # Set up logger
+    lt.logger(join(dir_save, 'log.txt'), lt.log.INFO)
+
+    # Define input files
+    file_pts_data = join(opencsp_code_dir(), 'test/data/sofast_common/aruco_corner_locations.csv')
```

Comment:
Should the data be moved to an 'input' subdirectory 'test/data/input/sofast_common/aruco_corner_locations.csv'?

Reply:
Yes... I remember thinking this was sloppy and remember thinking that I should totally change it. 
Measurement / expected data files are now in representative folders. All unit tests/examples updated.